package.json

```diff
@@ -6,7 +6,8 @@
     "install:pod": "cd ios && pod install",
     "postinstall": "patch-package",
     "postversion": "react-native-version",
-    "format:all": "prettier --write ./app/**/*.js"
+    "format:all": "prettier --write ./app/**/*.js",
+    "test": "jest test"
```

Comment:
@thetimbanks, @rparet 
Can we also include Jest testing on git pre-commit hooks on the same PR? It will help prevent folks from pushing code without adding unit tests and eventually prevent build failures on the pipeline.

`
"husky": {
    "hooks": {
      "pre-commit": "lint-staged && yarn test"
    }
  },
`

Reply:
@pathakparth I linked the code for a Github action that will run on all PRs in an above comment. I don't have permission to add actions to the project or I would just go ahead and add it. Once we get this merged we can get the action added and make it required on all PRs

Reply:
@thetimbanks 
Definitely it's a good idea to add it to Github actions, but I was referring to add the above code on line 17 of Package.json. It will perform "formatting and unit test" on git-commit hooks while developers trying to commit their code locally. This will prevent them from commit/pushing new/modified code for PR's if Jest testing is failing.

Anyway, there is a hack to skip this pre-commit hooks checks by using the following command "--no-verify" on git commit, but it helps in most of the cases if developers do not skip them.